### PR TITLE
libs: Add `envoy.code.check[glint]`

### PR DESCRIPTION
**Pull request by phlax**

### CODE REVIEW

File: envoy.code.check/envoy/code/check/abstract/checker.py [hunk 1]
```diff
@@ -91,6 +91,16 @@ def fs_directory_class(self) -> Type["_directory.ADirectory"]:
     def git_directory_class(self) -> Type["_directory.AGitDirectory"]:
         raise NotImplementedError
 
+    @cached_property
+    def glint(self) -> "abstract.AGlintCheck":
+        """Glint checker."""
+        return self.glint_class(self.directory, fix=self.fix)
```
Comment (phlax) [inline envoy.code.check/envoy/code/check/abstract/checker.py:97]: this doesnt implement any fixes - not sure if that matters

**Comment by phlax** [comment]
WIPping while i address some blocking issues and add some optimizations

File: envoy.code.check/envoy/code/check/abstract/glint.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        if not files:
+            return set()
+        # This is a workaround for a blocking issue sending
+        # very large cli args to a subprocess.
+        # TODO: generalize this solution in directory.grep
+        if len(files) < 1000:
+            return await self.directory.grep(
+                ["-lP", r"^\t",
+                 *await self.files])
+        return (
```
Comment (phlax) [inline envoy.code.check/envoy/code/check/abstract/glint.py:67]: these lines are not currently covered, but i want to move this to aio.core.directory - so will shift and address tests in #352

File: envoy.code.check/envoy/code/check/abstract/glint.py [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+            return set()
+        # This is a workaround for a blocking issue sending
+        # very large cli args to a subprocess.
+        # TODO: generalize this solution in directory.grep
+        if len(files) < 1000:
+            return await self.directory.grep(
+                ["-lE",
+                 "[[:blank:]]$",
+                 *await self.files])
+        return (
```
Comment (phlax) [inline envoy.code.check/envoy/code/check/abstract/glint.py:94]: ditto

**Review by kfaseela — APPROVED** [review]

**Pull request merged**